flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -41,19 +41,19 @@ echo "Flink distribution directory: $FLINK_DIR"
 
 # Template for adding a test:
 
-# run_test "<description>" "$END_TO_END_DIR/test-scripts/<script_name>"
+# run_test "<description>" "$END_TO_END_DIR/test-scripts/<script_name>" ["skip_check_exceptions"]
```

Comment:
I think we should add at least comments about what the consequences are for skipping the exceptions check / in what situations you would want to skip exception checking completely.